src/scss/components/checkbox/_index.scss

```diff
@@ -120,11 +120,11 @@ hx-checkbox-control {
 
     &:checked,
     &:indeterminate {
-      color: $cyan-900;
+      color: var(--hxCheckbox-indeterminate, $cyan-900);
```

Comment:
please change to var(--hxCheckbox-indeterminate-color, $cyan-900);

Reply:
Changes have been done, Please review